src/test/java/tictactoe/PartyCreatorTest.java

```diff
@@ -0,0 +1,14 @@
+package tictactoe;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PartyCreatorTest {
+    @Test
+    public void itCreatesAParty() {
+        PartyCreator partyCreator = new PartyCreator();
+        Party party = partyCreator.newParty(null, 3, GameTypes.COMPUTER_VS_COMPUTER);
+        assertThat(party).isInstanceOf(Party.class);
```

Comment:
Maybe a better assertion is to check that party isn't null here?
We can see in the test that it is a `Party` type, and Java wouldn't compile if something different was returned.